src/cloudant/design_document.py

```diff
@@ -216,6 +306,24 @@ def iterviews(self):
         for view_name, view in iteritems_(self.views):
             yield view_name, view
 
+    def iterindexes(self):
+        """
+        Provides a way to iterate over the locally cached DesignDocument
+        indexes dictionary.
+
+        For example:
+
+        .. code-block:: python
+
+            for index_name, search_func in ddoc.iterindexes():
+                # Perform search index processing
+
+        :returns: Iterable containing search index name and associated
+            search function object
```

Comment:
This doc block refers to search indexes. I think the example could still be for search indexes, but the returns statement ought to be corrected.